public/app/types/unified-alerting-dto.ts

```diff
@@ -23,6 +23,10 @@ type GrafanaAlertStateReason = ` (${string})` | '';
 
 export type GrafanaAlertStateWithReason = `${GrafanaAlertState}${GrafanaAlertStateReason}`;
 
+export function isGrafanaAlertState(state: string): state is GrafanaAlertState {
```

Comment:
Maybe `containsGrafanaAlertState` name makes more sense in this case?

Reply:
This method is a type guard which ensures that the `state` string is actually `GrafanaAlertState`. 
I think it's a common convention to call type guards as `is[TypeName]`